hpaction/hotdocs-data/Master.cmp

```diff
@@ -106,18 +119,112 @@ First, you have to complete this sentence: «.b»“My case is good and worthwhi
+			<hd:prompt>Hair color</hd:prompt>
+			<hd:fieldWidth widthType="calculated"/>
+		</hd:text>
+		<hd:text name="Served person landlord height TE">
+			<hd:prompt>Approximate height</hd:prompt>
+			<hd:fieldWidth widthType="calculated"/>
+		</hd:text>
+		<hd:text name="Served person landlord sex TE" maxChars="25">
+			<hd:prompt>Sex</hd:prompt>
+			<hd:fieldWidth widthType="calculated"/>
+		</hd:text>
+		<hd:text name="Served person landlord skin color TE">
+			<hd:prompt>Skin color</hd:prompt>
+			<hd:fieldWidth widthType="calculated"/>
+		</hd:text>
+		<hd:text name="Served person landlord weight TE">
+			<hd:prompt>Approximate weight</hd:prompt>
+			<hd:fieldWidth widthType="calculated"/>
 		</hd:text>
 		<hd:text name="Served person management company TE" warnIfUnanswered="false">
-			<hd:prompt>Name of person you handed the documents to
-«.i»(If you  or someone else delivers the documents by hand, you’ll need to write down the name of the person you gave them to and the address where you did that.  If you do not know yet who will receive them, you can leave this blank.)«.ie»</hd:prompt>
+			<hd:prompt>Name of person you handed the documents to</hd:prompt>
+		</hd:text>
+		<hd:text name="Served person mgmt co age TE">
+			<hd:prompt>Approximate age</hd:prompt>
+			<hd:fieldWidth widthType="calculated"/>
+		</hd:text>
+		<hd:text name="Served person mgmt co hair color TE">
+			<hd:prompt>Hair color</hd:prompt>
+			<hd:fieldWidth widthType="calculated"/>
+		</hd:text>
+		<hd:text name="Served person mgmt co height TE">
+			<hd:prompt>Approximate height</hd:prompt>
+			<hd:fieldWidth widthType="calculated"/>
+		</hd:text>
+		<hd:text name="Served person mgmt co sex TE" maxChars="25">
+			<hd:prompt>Sex</hd:prompt>
+			<hd:fieldWidth widthType="calculated"/>
+		</hd:text>
+		<hd:text name="Served person mgmt co skin color TE">
+			<hd:prompt>Skin color</hd:prompt>
+			<hd:fieldWidth widthType="calculated"/>
+		</hd:text>
+		<hd:text name="Served person mgmt co weight TE">
+			<hd:prompt>Approximate weight</hd:prompt>
+			<hd:fieldWidth widthType="calculated"/>
+		</hd:text>
+		<hd:text name="Served person mmco DHPD age TE">
+			<hd:prompt>Approximate age</hd:prompt>
+			<hd:fieldWidth widthType="calculated"/>
+		</hd:text>
+		<hd:text name="Served person mmco DHPD hair color TE">
+			<hd:prompt>Hair color</hd:prompt>
+			<hd:fieldWidth widthType="calculated"/>
+		</hd:text>
+		<hd:text name="Served person mmco DHPD height TE">
+			<hd:prompt>Approximate height</hd:prompt>
+			<hd:fieldWidth widthType="calculated"/>
+		</hd:text>
+		<hd:text name="Served person mmco DHPD sex TE" maxChars="25">
+			<hd:prompt>Sex</hd:prompt>
+			<hd:fieldWidth widthType="calculated"/>
+		</hd:text>
+		<hd:text name="Served person mmco DHPD skin color TE">
+			<hd:prompt>Skin color</hd:prompt>
+			<hd:fieldWidth widthType="calculated"/>
+		</hd:text>
+		<hd:text name="Served person mmco DHPD weight TE">
+			<hd:prompt>Approximate weight</hd:prompt>
```

Comment:
Some of these personal details (weight, hair/skin color, age, height) don't seem to be asked in the actual interview, but they _are_ fields on the generated PDF:

> ![description_of_person](https://user-images.githubusercontent.com/124687/48740269-cdc4ba80-ec24-11e8-8776-72b2d6e58cbf.png)